.github/workflows/test-go1.10.yaml

```diff
@@ -30,15 +30,21 @@ jobs:
       with:
         path: /var/cache/apt
         key: var-cache-apt-{{ hashFiles('**/debian/control') }}
+    - name: Run "apt update"
```

Comment:
Why is this moved out to a separate step? On a cache hit you avoid this entire section.

Reply:
Unfortunately the archive/ppa changes all the time so "apt install foo" will not work after a couple of days because the cache has v1.22 but the archive has moved on to 1.23 but the cache was not updated because nothing in debian/control has changed.

Reply:
Actually I think this is needed because of a hash key collision somehow